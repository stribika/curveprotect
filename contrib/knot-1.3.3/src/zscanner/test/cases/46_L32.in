$ORIGIN	.
$TTL	1

; OK
@	L32	0	0.0.0.0		; The simplest case
@	L32	65535	255.255.255.255	; The maximal case
@	L32	\# 6 000000000000	; Hexadecimal rdata
@	TYPE105	\# 6 000000000000	; TYPE + Hexadecimal rdata
@	TYPE105	0	0.0.0.0		; TYPE
@	l32	0	0.0.0.0		; Type in lower-case

; KO
@	L32
@	L32				; Empty rdata
@	L32	\# 0			; Hex empty rdata
@	L32	\#			; Missing hex length
@	L32	65536	0.0.0.0		; Too big preference

@	L32	0	0.0.0.256	; 8-bit overflow
@	L32	0	0.0.0		; Short address
@	L32	0	0.0.0.0.0	; Long address
@	L32	0	0.0.0.x		; Bad character
@	L32	0	0.0.0.0 x	; Unexpected item
@	L32	\# 7 000000000000 00	; Too long rdata
@	L32	\# 7 000000000000	; Bad rdata length
