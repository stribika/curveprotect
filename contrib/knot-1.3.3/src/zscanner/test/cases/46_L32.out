OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0069
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0069
RDATA=FFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0069
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0069
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0069
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0069
RDATA=000000000000
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER16_OVERFLOW
------
WARNG=ZSCANNER_EBAD_IPV4
------
WARNG=ZSCANNER_EBAD_IPV4
------
WARNG=ZSCANNER_EBAD_IPV4
------
WARNG=ZSCANNER_EBAD_ADDRESS_CHAR
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
